fenix/app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt

```diff
@@ -569,9 +597,11 @@ class BrowserFragment : BaseBrowserFragment(), UserInteractionHandler {
                     isNewCollection -> {
                         R.string.create_collection_tabs_saved_new_collection
                     }
+
                     tabSize > 1 -> {
                         R.string.create_collection_tabs_saved
                     }
+
```

Comment:
Let's undo this and avoid adding unrelated changes

Reply:
Done